frontend/src/components/Filters/ModelFilter.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from "react";
+import _ from "lodash";
+
+class ModelFilter extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    filterModel(models) {
+        let FrequentistRestricted = [];
+        let FrequentistUnRestricted = [];
+        let Bayesian = [];
+        let BayesianModelAverage = [];
+
+        for (let i = 0; i < models.length; i++) {
```

Comment:
use a map instead of loop, etc. `{frequentist_restricted: FrequentistRestricted}`